brayns/engine/scene/ModelManager.cpp

```diff
@@ -79,24 +79,22 @@ class ModelFinder
 
 namespace brayns
 {
-ModelInstance &ModelManager::addModel(ModelLoadParameters params, std::unique_ptr<Model> model)
+ModelInstance *ModelManager::addModel(std::unique_ptr<Model> model)
 {
     std::vector<std::unique_ptr<Model>> models;
     models.push_back(std::move(model));
-    auto instances = addModels(std::move(params), std::move(models));
-    return *(instances.front());
+    auto instances = addModels(std::move(models));
+    return instances.front();
```

Comment:
I would do the inverse, having addModels calling addModel and avoid creating the vector for single model.

Reply:
Addressed